### DISCUSSION TITLE
Please help!

### DISCUSSION THREAD

**Discussion by Jennywoof** [keiwando/toonsquid#591] [Q&A]
Hello! I was wondering how to do a turbulence displace effect on my animation? I can't figure out a way to do it on toonsquid and I'm very lost :')

**Comment by FoxeyeStudios** [comment]
There isn’t any way to do that on ToonSquid. At least not yet… To do that, you’d need to make your animation, export, and then import it into another program that does allow you to use a turbulence displace filter.
